### PR TITLE
Leaderboard: Hit the API

### PR DESCRIPTION
**Pull request by Whoistolu**
# Leaderboard: Hit the API
## In this milestone, I:
- Created a new game with the name of my choice by using the API.
- Implemented the "Refresh" button (receiving data from the API and parsing the JSON).
- Implemented the form "Submit" button (sending data to the API).
- Use `async` and `await` JavaScript features to consume the API.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Whoistolu, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ A JavaScript project for the Leaderboard list app, using webpack and ES6 feature
 
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-![screenshot](img/leaderboard.png)
+![screenshot](img/api.png)
 
 ## Built With
```
Comment (mke2111) [inline README.md:11]: - [ ] Within the readme file, add clear instructions and the necessary run commands to run your app's page.
You have implemented the POST and GET functionality of the app excellently, however I had to run `npm build` before I could interact with your app successfully. 
- Add `npm i` and `npm build` within the instructions of getting started, and then clearly indicate which index.html file should be opened.

Comment (Whoistolu) [reply README.md:11]: Hello @mke2111,
Thank you for the feedback.
I have implemented all changes.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Whoistolu, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**